sql/core/src/test/scala/org/apache/spark/sql/execution/WholeStageCodegenSuite.scala

```diff
@@ -325,4 +326,33 @@ class WholeStageCodegenSuite extends QueryTest with SharedSQLContext {
       checkAnswer(groupByWithId, Seq(Row(1, 2, 0), Row(1, 2, 0)))
     }
   }
+
+  test("WholeStageCodegen does not work properly for LocalTableScanExec") {
```

Comment:
Could you use `SPARK-28520` prefix since this is a bug fix?

Reply:
Thanks. I added the prefix.